components/uid.rst

```diff
@@ -63,6 +63,20 @@ to create each type of UUID::
     // It's defined in http://gh.peabody.io/uuidv6/
     $uuid = Uuid::v6(); // $uuid is an instance of Symfony\Component\Uid\UuidV6
 
+    // UUID version 7 features a time-ordered value field derived from the widely implemented and well known Unix Epoch timestamp source,
+    // the number of milliseconds seconds since midnight 1 Jan 1970 UTC, leap seconds excluded.
+    // As well as improved entropy characteristics over versions 1 or 6.
```

Comment:
```suggestion
    // UUID version 7 features a time-ordered value field derived from the widely implemented and well known
    // Unix Epoch timestamp source, the number of milliseconds seconds since midnight 1 Jan 1970 UTC, leap seconds excluded.
    // As well as improved entropy characteristics over versions 1 or 6.
```